dist/protractor-testability-plugin.js

```diff
@@ -45,6 +45,10 @@ return {
                         };
                     }
                 };
+
+                window.getAllAngularTestabilities = function() {
```

Comment:
Very good catch. Seems to have a lot of value independently of the `readyFor` functionality we are discussing. If it doesn't bother you a lot, could you create a separate PR for this?